src/startServerAndCreateNextHandler.ts

```diff
@@ -50,6 +55,8 @@ function startServerAndCreateNextHandler<
       if (httpGraphQLResponse.body.kind === 'complete') {
         res.send(httpGraphQLResponse.body.string);
```

Comment:
After supporting incremental responses we have to move `res.end()` to here so we don't end the response prematurely:
```suggestion
        res.send(httpGraphQLResponse.body.string);
        res.end();
```